tapestry.php

```diff
@@ -383,4 +384,42 @@ function tapestry_tool_log_event() {
     $analytics->log($_POST);
 
     wp_die();
-}
\ No newline at end of file
+}
+
+// Plugin Update
+
+function accordion_update() {
+    $query_args = array(
+        'post_type' => 'tapestry_node',
+        'meta_query' => array(
+            array(
+               'key'     => 'tapestry_node_data',
+               'value'   => '"mediaType";s:9:"accordion"',
+               'compare' => 'LIKE',
+            ),
+        ),
+    );
+    $query = new WP_Query( $query_args );
+    while ( $query->have_posts() ) {
+        $query->the_post();
+        $post_id = get_the_ID();
+        $node_meta_data = get_post_meta($post_id,'tapestry_node_data', true);
+        if (isset($node_meta_data->mediaType) && $node_meta_data->mediaType == 'accordion') {
+            $node_meta_data->mediaType = 'multi-content';
+            $node_meta_data->presentationStyle = 'accordion';
+            update_post_meta( $post_id, 'tapestry_node_data',  $node_meta_data);
+        }
+    }
+}
+register_activation_hook( __FILE__, 'accordion_update' );
+
+function tapestry_plugin_update() {
+    global $TAPESTRY_VERSION_NUMBER;
+    $installed_version = get_site_option('tapestry_version');
+    if ( $installed_version != $TAPESTRY_VERSION_NUMBER) {
```

Comment:
It might work best if we check the specific version number here instead to see if `tapestry_version` is older than the current version. That way, this wouldn't run on every version update.

Reply:
Updated. Please let me know if this was what you're looking for.